src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -239,10 +239,11 @@ export const BorrowInput = ({
     history.push(BorrowRoutePaths.Sweep)
   }, [history, isSweepNeeded])
 
-  const depositAssetSelectComponent = useMemo(() => {
+  const collateralAssetSelectComponent = useMemo(() => {
     return (
       <TradeAssetSelect
         assetId={collateralAssetId}
+        // no-op
```

Comment:
?

Reply:
Actually a no-op, we can't change the collateral for borrows